config/clients/js/template/common.mustache

```diff
@@ -169,15 +169,13 @@ export async function attemptHttpRequest<B, R>(
 /**
  * creates an axios request function
  */
-export const createRequestFunction = function (axiosArgs: RequestArgs, axiosInstance: AxiosInstance, configuration: Configuration, credentials?: Credentials) {
+export const createRequestFunction = function (axiosArgs: RequestArgs, axiosInstance: AxiosInstance, configuration: Configuration, credentials: Credentials) {
   configuration.isValid();
 
   const retryParams = axiosArgs.options?.retryParams ? axiosArgs.options?.retryParams : configuration.retryParams;
   const maxRetry:number = retryParams ? retryParams.maxRetry : 0;
   const minWaitInMs:number = retryParams ? retryParams.minWaitInMs : 0;
-  if (!credentials) {
-    credentials = Credentials.init(configuration);
-  }
+
```

Comment:
This is a breaking change, no? It's doing credential initialization and configuration: https://github.com/openfga/js-sdk/blob/main/credentials/credentials.ts#L29-L56